ext/gtest/CMakeLists.txt

```diff
@@ -15,11 +14,4 @@ if(IVW_UNITTESTS OR IVW_INTEGRATION_TESTS)
         target_compile_definitions(gtest INTERFACE -DGTEST_LINKED_AS_SHARED_LIBRARY=1)
         #target_compile_definitions(gtest_main PRIVATE -DGTEST_LINKED_AS_SHARED_LIBRARY=0)
     endif()
-
-    ivw_register_license_file(NAME "Google C++ Testing Framework" TARGET gtest MODULE Core
```

Comment:
Is this added somewhere else?

Reply:
Yes, in core, unfortunatly it does not work in that spot...